entries/c/circles.life.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Circles Life": {
+    "domain": "circles.life",
+    "contact": {
+      "facebook": "CirclesLifeSG",
+      "twitter": "CirclesLifeSG",
```

Comment:
That Twitter account doesn't seem active, as they haven't posted anything for a year.
```suggestion
```